content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,401 @@
+    <br />
+
+5. The installer installs to the MAAS machine(s) and uses the configuration file to build a new cluster to host the PCG application. 
+
+<br />
+
+
+<Tabs>
+
+<Tabs.TabPane tab="Install PCG" key="install_pcg"> 
+
+
+## Install the Gateway
+
+The following steps will guide you to install the PCG. 
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. If you have Single or Social Sign-On (SSO) enabled, you will need to use or create a local non-SSO tenant admin account in Palette and use the credentials for that account in step **7**.
+
+<WarningBox>
+
+The installer does not work with SSO or Social sign on credentials. You must use a username and password from a local tenant admin account in Palette to deploy the PCG. After the PCG is configured and functioning, this local account is no longer used to keep the PCG connected to Palette, so you can disable the account if desired.
+
+</WarningBox>
+
+
+3. Navigate to the **Main Menu** and select **Tenant Settings > Private Cloud Gateway**.
+
+
+4. Click the **Create Private Cloud Gateway** button and select **MAAS**. Private Gateway installation instructions are displayed.
+
+
+5. Note the pairing code displayed in the Instructions section of the page. You will input this code when you use the installer. 
+
+
+6. To invoke the installer, copy the following code snippet to your terminal.
+    <br />
+
+    ```bash
+    docker run -it --rm \
+    --net=host \
+    --volume /var/run/docker.sock:/var/run/docker.sock \
+    --volume /tmp:/opt/spectrocloud \
+    gcr.io/spectro-images-public/release/spectro-installer:1.0.12
+    ```
+
+7. When prompted, enter the pairing code and information listed in each of the following tables. The installer will generate the gateway configuration file. 
+    <br />
+
+
+#### Palette Parameters
+
+|**Parameter**       | **Description**|
+|:-----------------------------|---------------|
+|**Install Type**| Choose **Private Cloud Gateway**. <br />You can change your selection with the up or down keys.|
+|**Cloud Type**| Choose MAAS.|
+|**Name** | Enter a custom name for the PCG. Example: ``maas-pcg-1``.|
+|**Endpoint** |Enter the Palette endpoint URL. When using the Palette SaaS service, enter ``https://console.spectrocloud.com``. When using a dedicated instance of Palette, enter the URL for that instance. |
+|**Username** |Enter your Palette username. This is your sign-in email address. Example: ``user1@company.com``. |
+|**Password** |Enter your Palette Password. This is your sign-in password.|
+|**Private Cloud Gateway** |Enter the pairing code you noted from the instructions page in step **5**. |
```

Comment:
@ritawatson change `**Private Cloud Gateway**` to `**Pairing Code**`

Reply:
fixed typo